userguide/sharing.rst

```diff
@@ -1852,42 +1852,35 @@ initiator.
-   | Authorized Networks | string    | Network addresses that can use this initiator. Use :literal:`ALL` or list network    |
-   |                     |           | addresses with a `CIDR                                                               |
-   |                     |           | <https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing>`__ mask. Separate     |
-   |                     |           | multiple addresses with a space: :samp:`192.168.2.0/24 192.168.2.1/12`.              |
-   |                     |           |                                                                                      |
-   +---------------------+-----------+--------------------------------------------------------------------------------------+
-   | Comment             | string    | Notes or a description of the initiator.                                             |
-   |                     |           |                                                                                      |
-   +---------------------+-----------+--------------------------------------------------------------------------------------+
-
-
-In the example shown in
-:numref:`Figure %s <iscsi_initiator_conf_sample_fig>`,
-two groups are created. Group 1 allows connections from any
-initiator on any network. Group 2 allows connections from any
-initiator on the *10.10.1.0/24* network. Click |ui-options| on an
-initiator entry to display its :guilabel:`Edit` and :guilabel:`Delete`
-buttons.
-
-.. note:: Attempting to delete an initiator causes a warning that
-   indicates if any targets or target/extent mappings depend upon the
-   initiator. Confirming the delete causes these to be deleted also.
-
-
-.. _iscsi_initiator_conf_sample_fig:
-
-.. figure:: images/sharing-block-iscsi-initiators-example.png
-
-   Sample iSCSI Initiator Configuration
+   +----------------------+-----------+--------------------------------------------------------------------------------------+
+   | Setting              | Value     | Description                                                                          |
+   |                      |           |                                                                                      |
+   +======================+===========+======================================================================================+
+   | Allow All Initiators | checkbox  | Set to accept all detected initiators. Disables all other initiator fields.          |
+   +----------------------+-----------+--------------------------------------------------------------------------------------+
+   | Connected Initiators | string    | Initiators currently connected to the system. Shown in IQN format with an IP         |
+   |                      |           | address. Set initiators and click an :menuselection:` --> (Arrow)` to add the        |
```

Comment:
If we need an arrow to show here, let's define a substitution to use rather than overloading the menuselection role.